ansible/roles/debops.dovecot/tasks/main.yml

```diff
@@ -19,6 +19,27 @@
   register: dovecot__register_packages
   until: dovecot__register_packages is succeeded
 
+- name: Make sure Ansible local facts directory exists
+  file:
+    dest: '/etc/ansible/facts.d'
+    state: 'directory'
+    owner: 'root'
+    group: 'root'
```

Comment:
You should remove the `owner` and `group` keywords from the tasks, since the role will be executed as `root` anyway.